snowpack/src/util.ts

```diff
@@ -269,15 +271,27 @@ export function findMatchingAliasEntry(
   ) {
     return undefined;
   }
-  const foundEntry = Object.entries(config.alias).find(([fromAlias]) => spec.startsWith(fromAlias));
-  if (!foundEntry) {
-    return undefined;
+
+  for (const [from, to] of Object.entries(config.alias)) {
+    let foundType: 'package' | 'path' | '' = '';
```

Comment:
/cc @drwpow `'' = ''` as a type seems fishy, I would have liked to have cleaned this up a bit before merging. Will create a quick PR to follow up.